src/pages/Settings/components/CategorySettings.tsx

```diff
@@ -82,6 +107,18 @@ const CategorySettings: React.FunctionComponent<Props> = ({
 			dataIndex: 'description',
 			editable: true,
 		},
+		{
+			title: '',
+			dataIndex: 'operation',
+			render: (text, record: Category) => (
+				<Popconfirm
```

Comment:
(status === 'saving') ? <Spinner/> . : <Popconfirm..../>

Reply:
Yeah I think we can left it later since we are using relatively fast network